### ISSUE TITLE
Installation issue with release 2.1.2+

### ISSUE THREAD

**Issue by gvisniuc** [twingly/twingly-search-api-python#51]
Hi, whenever I try install the package using `pip install twingly-search` for versions 2.1.2 and 2.1.3 I get the following error.

```
Collecting twingly-search==2.1.3
  Using cached twingly-search-2.1.3.tar.gz (9.4 kB)
    ERROR: Command errored out with exit status 1:
     command: /Users/george/anaconda3/bin/python -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/private/var/folders/wn/_zj563cn6k715p3k1q3jrq0c0000gn/T/pip-install-b266gkov/twingly-search/setup.py'"'"'; __file__='"'"'/private/var/folders/wn/_zj563cn6k715p3k1q3jrq0c0000gn/T/pip-install-b266gkov/twingly-search/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /private/var/folders/wn/_zj563cn6k715p3k1q3jrq0c0000gn/T/pip-install-b266gkov/twingly-search/pip-egg-info
         cwd: /private/var/folders/wn/_zj563cn6k715p3k1q3jrq0c0000gn/T/pip-install-b266gkov/twingly-search/
    Complete output (5 lines):
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/private/var/folders/wn/_zj563cn6k715p3k1q3jrq0c0000gn/T/pip-install-b266gkov/twingly-search/setup.py", line 17, in <module>
        with open('requirements.txt') as fp:
    FileNotFoundError: [Errno 2] No such file or directory: 'requirements.txt'
    ----------------------------------------
ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
```

Tried using python 3.6 and 3.7 both on my OSX and inside a fresh docker container.

**Comment by walro** [comment]
Thanks for reporting, we will look into it.

**Comment by Pontus4** [comment]
The problem should be solved in 2.1.4. Thanks again for reporting!
